.mergify.yml

```diff
@@ -73,16 +73,16 @@ pull_request_rules:
 - actions:
     backport:
       branches:
-      - '3.12'
```

Comment:
We actually still have 3.12 on F28 and F29, so I think we should keep this around in case we need to backport anything for those releases until they aren't on that version. I don't recall the reason, but I had opted not to bring them to 3.13 when it was released.